### DISCUSSION TITLE
IT and ES translations

### DISCUSSION THREAD

**Discussion by gissimo** [pallets-eco/flask-security#872] [Ideas]
Hi all,

I'm available to start anew the Italian translation and complete the Spanish one.

If I start working on that, is a PR request going to be considered?

Best,

Giorgio

**Comment by jwag956** [comment]
Translation prs are also appreciated

**Comment by Efois** [comment]
> Hi all,
> 
> I'm available to start anew the Italian translation and complete the Spanish one.
> 
> If I start working on that, is a PR request going to be considered?
> 
> Best,
> 
> Giorgio

Hi.
I would like help in this. How could we do it together?
I
